tcl/org.testeditor.tcl.dsl/src/org/testeditor/tcl/dsl/Tcl.xtext

```diff
@@ -51,6 +49,22 @@ TestStep:
 	'-' contents+=StepContent* '.'?
 ;
 
+TestStepAssertion:
+	'-' 'assert' (negated?='!')? variableName=ID (comparator=Comparator contents+=StepContent)? '.'?
```

Comment:
`contents+=StepContent` is too generic as it allows: `StepContentText | StepContentElement | StepContentVariable`